### PR TITLE
Feature registration form

### PR DESCRIPTION
**Pull request by abhijeet2902**
Do check and let me Know only Single Component should be present under the name of RegistrationForm ,if more present then I have to delete my branch gadbad ho jaayegi

### CODE REVIEW

**Comment by aayushtailang1911** [comment]
good to go

File: amazon-clone/src/App.vue [hunk 1]
```diff
@@ -1,15 +1,19 @@
 <template>
-  <navbar-component />
```
Comment (aayushtailang1911) [inline amazon-clone/src/App.vue:2]: why there is space

File: amazon-clone/src/App.vue [hunk 2]
```diff
@@ -1,15 +1,19 @@
...
   
+  <navbar-component />
+ <router-view></router-view>
+
 </template>
 
 <script>
 import NavbarComponent from "./components/layout/NavbarComponent.vue";
+//import RegistrationForm from "./components/RegistrationForm.vue"
```
Comment (aayushtailang1911) [inline amazon-clone/src/App.vue:10]: remove this

File: amazon-clone/src/components/RegistrationForm.vue [hunk 3]
```diff
@@ -0,0 +1,112 @@
+<template>
+    <form @submit.prevent="toRegister" class="custom-form">
+       <div class="mb-3">
+        <label for="firstName" class="form-label">FirstName:</label>
```
Comment (aayushtailang1911) [inline amazon-clone/src/components/RegistrationForm.vue:4]: there should be space between first name

File: amazon-clone/src/components/RegistrationForm.vue [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+   <input type="text" class="form-control" id="firstName" v-model="user.firstName" required>
+</div>
+<div class="mb-3">
+        <label for="lastName" class="form-label">LastName:</label>
```
Comment (aayushtailang1911) [inline amazon-clone/src/components/RegistrationForm.vue:8]: same as above

File: amazon-clone/src/App.vue (hunk 2 above)
Comment (abhijeet2902) [reply amazon-clone/src/App.vue:10]: ok Sir

Comment (abhijeet2902) [reply amazon-clone/src/App.vue:10]: Done in my remote repo please approve this now

File: amazon-clone/src/App.vue (hunk 1 above)
Comment (abhijeet2902) [reply amazon-clone/src/App.vue:2]: I enclosed these under a div as per our screen share you asked me to remove that div hence the space ,no need to worry i removed it in my remote repo ,raising a new PR now with both the changes you suggested.

**Review by aayushtailang1911 — APPROVED** [review]
good

**Pull request closed without merging**